modules/gdpr_tasks/config/install/field.field.gdpr_task.gdpr_sar.sar_export_assets.yml

```diff
@@ -0,0 +1,26 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.gdpr_task.sar_export_assets
+    - gdpr_tasks.gdpr_task_type.gdpr_sar
+  module:
+    - file
+id: gdpr_task.gdpr_sar.sar_export_assets
```

Comment:
Would this be better as a base field? I think we've figured out how to do that at a bundle level? If it get's removed it'll cause lots of problems...